### ISSUE TITLE
No Debug Launch Configuration Options for react-native extension

### ISSUE THREAD

**Issue by Chuk116** [microsoft/vscode-react-native#2150]
I am having a similar issue that I can't quite figure out. I have the react-native tools extension downloaded on VSCode but when I click on the Add Configuration button I don't see any options for React Native. Additionally, when I click on the "Show all automatic debug configurations" in the Run And Debug side panel, I see the "React Native" option like this: 
<img width="1782" alt="Screenshot 2024-05-22 at 5 01 52 PM" src="https://github.com/microsoft/vscode-react-native/assets/28410610/bec59f3b-005c-4c99-a2aa-7100f7088c4f">
However, when I click the React Native option the dropdown is empty like this: 
<img width="1791" alt="Screenshot 2024-05-22 at 5 02 22 PM" src="https://github.com/microsoft/vscode-react-native/assets/28410610/6714f979-140a-47f5-acad-d1df3b1f95b4"> I feel like I have tried every solution that I have found on the web for related issues but nothing has worked so far. Any ideas for other possible solutions?

_Originally posted by @Chuk116 in https://github.com/microsoft/vscode-react-native/issues/985#issuecomment-2125742939_

**Comment by EzioLi01** [comment]
Hey @Chuk116 , thanks for reaching us. Which version of vscode have you used?

**Comment by EzioLi01** [comment]
Hey @Chuk116 , do you have any updates on this?

**Comment by EzioLi01** [comment]
Hey @Chuk116 , thanks for the details.
At your screenshot, your extension is not activated, this extension will be activated when vscode open a react-native project folder. If you don't have any specific demo, could you please try to create a sample project using `npx react-native init testApp`, and open it in vscode to check if extension can be activated?

**Comment by EzioLi01** [comment]
Hey, do you have chance to check it?

**Comment by Chuk116** [comment]
Yes sorry, I created the sample project but it doesn't seem to be activated within the sample project either: 

<img width="816" alt="Screenshot 2024-06-04 at 8 57 00 PM" src="https://github.com/microsoft/vscode-react-native/assets/28410610/b068ab42-2882-45de-8b4a-25615d7de686">

**Comment by EzioLi01** [comment]
> Yes sorry, I created the sample project but it doesn't seem to be activated within the sample project either:
> 
> <img alt="Screenshot 2024-06-04 at 8 57 00 PM" width="816" src="https://private-user-images.githubusercontent.com/28410610/336681459-b068ab42-2882-45de-8b4a-25615d7de686.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1NjY1MDEsIm5iZiI6MTcxNzU2NjIwMSwicGF0aCI6Ii8yODQxMDYxMC8zMzY2ODE0NTktYjA2OGFiNDItMjg4Mi00NWRlLThiNGEtMjU2MTVkN2RlNjg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA1NDMyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3ZjliNDk2NmM2MzI4M2FmZjczZDM0ZGIzODliNThjMDIwYzQ0ZGQ1YTZlMmVkYzg0Nzk2ZjM3YjRlMGI1YWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lMc7o31l2uSszJJCffA_0ljUs3rKxfFZbWePZWjeEp0">

I still cannot repro it, could you please show me a full screenshot or record a video for it? I will keep this issue opening, but I must say, since it's hardly to do any investigate if I cannot get some useful information. Thanks.

**Comment by EzioLi01** [comment]
Close this issue since no updates from customer currently, if any detail needs to update, please reopen this or create a new one. Thanks!

**Comment by lucassad** [comment]
I am also facing this issue, any fix?
